packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -34,6 +35,7 @@ interface Props {
   mapillaryCallback?: (id: string) => void;
   mapillaryKey?: string;
   setActiveLeg: SetActiveLegFunction;
+  showApproximatePrefixAccessLegs?: boolean;
```

Comment:
Sorry for the back and forth... how about renaming to `showApproximateTravelTime` for this component? (No need to say "access leg" in the prop because we are within that component.)